### ISSUE TITLE
inputs values and checkboxes cant be changed

### ISSUE THREAD

**Issue by StealthyDamn** [utatti/react-render-html#35]
when I render input value="something" or checkbox with checked="checked" it stays that ad cannot be changed, this behaviour is wrong in my opinion, it does not behave the right way forms should behave...

**Comment by luanfonceca** [comment]
I am facing it too, did you figure out a solution?
I will try to, somehow, create a state for the inputs to see if it works.

**Comment by hejrobin** [comment]
This is how React works; https://reactjs.org/docs/forms.html

**Comment by StealthyDamn** [comment]
No, react lets you set defVaule, there is no choice to make defValue with this component, so it is unsusable for forms with prefilled values...

Argument for implementing value="" to be defValue for input component is that this component is for using html in react and html does NOT work this way... So you are not enabling to us the use of HTML as __dangerouslySetHtml does... it should be replacemtnt for this but it is NOT because it does not let us use forms... but dangerously sethtml does...
________________________________
Od: Robin Grass <notifications@github.com>
Odesláno: 18. ledna 2018 13:32
Komu: utatti/react-render-html
Kopie: StealthyDamn; Author
Předmět: Re: [utatti/react-render-html] inputs values and checkboxes cant be changed (#35)


This is how React works; https://reactjs.org/docs/forms.html
